Latest OF release 0.9 not compiling

Hi,  Have tried several times today, Nov9th 2015, to get the latest release of, OF,  to compile on Ubuntu 15.10 without success, terminal output copied below.  Tried editing the, .compiler_flags, file in the debug folder to include the missing reference to the, boost,  folder but it seems to get overwritten. Best I can do as I don't know how the compiler_flags file is generated or even if I am on the right track. 




HOST_OS=Linux
HOST_ARCH=x86_64
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal openssl libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
HOST_OS=Linux
HOST_ARCH=x86_64
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal openssl libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
Compiling /home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/3d/ofMesh.cpp
g++ -g3  -Wall -std=c++14 -DGCC_HAS_REGEX -DOF_USING_GTK -DOF_USING_MPG123 -D_REENTRANT -pthread -I/usr/include/gstreamer-1.0 -I/usr/lib/x86_64-linux-gnu/gstreamer-1.0/include -I/usr/include/AL -I/usr/include/alsa -I/usr/include/GL -I/usr/include/libdrm -I/usr/include/gtk-3.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib/x86_64-linux-gnu/dbus-1.0/include -I/usr/include/gtk-3.0 -I/usr/include/gio-unix-2.0/ -I/usr/include/mirclient -I/usr/include/mircommon -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/harfbuzz -I/usr/include/pango-1.0 -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng12 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng12 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/home/bernie/of_v0.9.0_linux64_release/libs/fmodex/include -I/home/bernie/of_v0.9.0_linux64_release/libs/glfw/include -I/home/bernie/of_v0.9.0_linux64_release/libs/glfw/include/GLFW -I/home/bernie/of_v0.9.0_linux64_release/libs/kiss/include -I/home/bernie/of_v0.9.0_linux64_release/libs/poco/include -I/home/bernie/of_v0.9.0_linux64_release/libs/tess2/include -I/home/bernie/of_v0.9.0_linux64_release/libs/utf8cpp/include -I/home/bernie/of_v0.9.0_linux64_release/libs/utf8cpp/include/utf8 -I/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks -I/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/3d -I/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/math -I/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/sound -I/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/video -I/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/app -I/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/events -I/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/types -I/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/gl -I/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/communication -I/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/utils -I/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/graphics -DDEBUG  -MMD -MP -MF /home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/obj/Debug/libs/openFrameworks/3d/ofMesh.d -MT/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/obj/Debug/libs/openFrameworks/3d/ofMesh.o -o /home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/obj/Debug/libs/openFrameworks/3d/ofMesh.o -c /home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/3d/ofMesh.cpp
In file included from /home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/utils/ofLog.h:4:0,
                 from /home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/utils/ofUtils.h:7,
                 from /home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/3d/ofMesh.h:10,
                 from /home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/3d/ofMesh.cpp:1:
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworks/utils/ofFileUtils.h:8:32: fatal error: boost/filesystem.hpp: No such file or directory
compilation terminated.
makefileCommon/compile.core.mk:239: recipe for target '/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/obj/Debug/libs/openFrameworks/3d/ofMesh.o' failed
make[1]: *** [/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/obj/Debug/libs/openFrameworks/3d/ofMesh.o] Error 1
makefileCommon/compile.core.mk:213: recipe for target 'Debug' failed
make: *** [Debug] Error 2
there has been a problem compiling Debug OF library
please report this problem in the forums

Good morning, further to last comment ..... tried to compile polgonExample, and it failed to compile so went to check through the installed dependencies, and found,
libsndfile is libsndfile1 in my system,
the libopencv-dev hadn't been installed,
libassimp and Poco were also missing.
Installed all except Poco using synaptic recompiled and finished the compilation. Again the example failed to compile.
Have just installed Poco dependencies and compiled OF again. Terminal output belowbernie@Bernie7:~/of_v0.9.0_linux64_release/examples/graphics/polygonExample$ make
HOST_OS=Linux
HOST_ARCH=x86_64
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal openssl libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
Compiling OF library for Release
make[1]: Entering directory '/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/project'
HOST_OS=Linux
HOST_ARCH=x86_64
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal openssl libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
HOST_OS=Linux
HOST_ARCH=x86_64
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal openssl libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
HOST_OS=Linux
HOST_ARCH=x86_64
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal openssl libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
Done!
make[1]: Leaving directory '/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/project'


Compiling polygonExample for Release
make[1]: Entering directory '/home/bernie/of_v0.9.0_linux64_release/examples/graphics/polygonExample'
HOST_OS=Linux
HOST_ARCH=x86_64
checking pkg-config libraries:   cairo zlib gstreamer-app-1.0 gstreamer-1.0 gstreamer-video-1.0 gstreamer-base-1.0 libudev freetype2 fontconfig sndfile openal openssl libpulse-simple alsa gl glu glew gtk+-3.0 libmpg123 
Linking bin/polygonExample for linux64
g++ -o bin/polygonExample  obj/linux64/Release/src/ofApp.o obj/linux64/Release/src/main.o  /home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a   -Wl,-rpath=./libs:./bin/libs -Wl,--as-needed -Wl,--gc-sections -L/home/bernie/of_v0.9.0_linux64_release/libs/fmodex/lib/linux64/ -lfmodex /home/bernie/of_v0.9.0_linux64_release/libs/glfw/lib/linux64/libglfw3.a /home/bernie/of_v0.9.0_linux64_release/libs/kiss/lib/linux64/libkiss.a /home/bernie/of_v0.9.0_linux64_release/libs/tess2/lib/linux64/libtess2.a  /home/bernie/of_v0.9.0_linux64_release/libs/poco/lib/linux64/libPocoNetSSL.a /home/bernie/of_v0.9.0_linux64_release/libs/poco/lib/linux64/libPocoNet.a /home/bernie/of_v0.9.0_linux64_release/libs/poco/lib/linux64/libPocoCrypto.a /home/bernie/of_v0.9.0_linux64_release/libs/poco/lib/linux64/libPocoUtil.a /home/bernie/of_v0.9.0_linux64_release/libs/poco/lib/linux64/libPocoJSON.a /home/bernie/of_v0.9.0_linux64_release/libs/poco/lib/linux64/libPocoXML.a /home/bernie/of_v0.9.0_linux64_release/libs/poco/lib/linux64/libPocoFoundation.a  -L/usr/lib/x86_64-linux-gnu -lz -lgstapp-1.0 -lgstvideo-1.0 -lgstbase-1.0 -lgstreamer-1.0 -ludev -lfontconfig -lfreetype -lsndfile -lopenal -lssl -lcrypto -lpulse-simple -lpulse -lasound -lGLEW -lGLU -lGL -lgtk-3 -lgdk-3 -lpangocairo-1.0 -lpango-1.0 -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lglib-2.0 -lmpg123  -lglut -lX11 -lXrandr -lXxf86vm -lXi -lXcursor -ldl -lpthread -lfreeimage -lrtaudio -lboost_filesystem -lboost_system
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofUtils.o): In function `ofLaunchBrowser(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, bool)':
ofUtils.cpp:(.text+0x8fac): undefined reference to `Poco::URI::URI(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
ofUtils.cpp:(.text+0x8fea): undefined reference to `Poco::URI::setQuery(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
ofUtils.cpp:(.text+0x9016): undefined reference to `Poco::URI::toString[abi:cxx11]() const'
ofUtils.cpp:(.text+0x9198): undefined reference to `Poco::URI::toString[abi:cxx11]() const'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofURLFileLoader.o): In function `ofURLFileLoaderImpl::threadedFunction()':
ofURLFileLoader.cpp:(.text+0x4918): undefined reference to `Poco::Thread::setName(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
ofURLFileLoader.cpp:(.text+0x64d6): undefined reference to `Poco::SystemException::SystemException(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, int)'
ofURLFileLoader.cpp:(.text+0x659e): undefined reference to `Poco::SystemException::SystemException(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, int)'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofURLFileLoader.o): In function `ofURLFileLoaderImpl::handleRequest(ofHttpRequest)':
ofURLFileLoader.cpp:(.text+0x7426): undefined reference to `Poco::URI::URI(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
ofURLFileLoader.cpp:(.text+0x7438): undefined reference to `Poco::URI::getPathAndQuery[abi:cxx11]() const'
ofURLFileLoader.cpp:(.text+0x7463): undefined reference to `Poco::Net::HTTPMessage::HTTP_1_1[abi:cxx11]'
ofURLFileLoader.cpp:(.text+0x7468): undefined reference to `Poco::Net::HTTPRequest::HTTP_GET[abi:cxx11]'
ofURLFileLoader.cpp:(.text+0x7478): undefined reference to `Poco::Net::HTTPRequest::HTTPRequest(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
ofURLFileLoader.cpp:(.text+0x749c): undefined reference to `Poco::Net::NameValueCollection::add(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
ofURLFileLoader.cpp:(.text+0x750d): undefined reference to `Poco::Net::HTTPSClientSession::HTTPSClientSession(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned short)'
ofURLFileLoader.cpp:(.text+0x77eb): undefined reference to `Poco::Net::HTTPClientSession::HTTPClientSession(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned short)'
ofURLFileLoader.cpp:(.text+0x7985): undefined reference to `Poco::Exception::displayText[abi:cxx11]() const'
ofURLFileLoader.cpp:(.text+0x7a94): undefined reference to `Poco::Exception::displayText[abi:cxx11]() const'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofURLFileLoader.o): In function `ofURLFileLoaderImpl::ofURLFileLoaderImpl()':
ofURLFileLoader.cpp:(.text+0x8dae): undefined reference to `Poco::Net::Context::Context(Poco::Net::Context::Usage, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, Poco::Net::Context::VerificationMode, int, bool, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
ofURLFileLoader.cpp:(.text+0x92a0): undefined reference to `Poco::Exception::displayText[abi:cxx11]() const'
ofURLFileLoader.cpp:(.text+0x93d3): undefined reference to `Poco::Exception::displayText[abi:cxx11]() const'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofURLFileLoader.o): In function `Poco::ScopedLock<Poco::FastMutex>::~ScopedLock()':
ofURLFileLoader.cpp:(.text._ZN4Poco10ScopedLockINS_9FastMutexEED2Ev[_ZN4Poco10ScopedLockINS_9FastMutexEED5Ev]+0x5c): undefined reference to `Poco::SystemException::SystemException(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, int)'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofImage.o): In function `bool loadImage<unsigned short>(ofPixels_<unsigned short>&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)':
ofImage.cpp:(.text+0x27d3): undefined reference to `Poco::URI::URI(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofImage.o): In function `bool loadImage<unsigned char>(ofPixels_<unsigned char>&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)':
ofImage.cpp:(.text+0x2ca3): undefined reference to `Poco::URI::URI(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofImage.o): In function `bool loadImage<float>(ofPixels_<float>&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)':
ofImage.cpp:(.text+0x3443): undefined reference to `Poco::URI::URI(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofThread.o): In function `ofThread::run()':
ofThread.cpp:(.text+0x2ac): undefined reference to `Poco::Exception::displayText[abi:cxx11]() const'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofThread.o): In function `ofThread::ofThread()':
ofThread.cpp:(.text+0x768): undefined reference to `Poco::Thread::setName(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofThread.o): In function `ofThread::startThread(bool)':
ofThread.cpp:(.text+0xf6d): undefined reference to `Poco::SystemException::SystemException(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, int)'
ofThread.cpp:(.text+0xfe0): undefined reference to `Poco::SystemException::SystemException(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, int)'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofThread.o): In function `ofThread::getThreadName[abi:cxx11]() const':
ofThread.cpp:(.text+0x1222): undefined reference to `Poco::SystemException::SystemException(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, int)'
ofThread.cpp:(.text+0x1294): undefined reference to `Poco::SystemException::SystemException(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, int)'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofThread.o): In function `ofThread::threadedFunction()':
ofThread.cpp:(.text+0x1581): undefined reference to `Poco::SystemException::SystemException(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, int)'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/lib/linux64/libopenFrameworks.a(ofThread.o):ofThread.cpp:(.text+0x15f4): more undefined references to `Poco::SystemException::SystemException(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, int)' follow
collect2: error: ld returned 1 exit status
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/project/makefileCommon/compile.project.mk:381: recipe for target 'bin/polygonExample' failed
make[1]: *** [bin/polygonExample] Error 1
make[1]: Leaving directory '/home/bernie/of_v0.9.0_linux64_release/examples/graphics/polygonExample'
/home/bernie/of_v0.9.0_linux64_release/libs/openFrameworksCompiled/project/makefileCommon/compile.project.mk:126: recipe for target 'Release' failed
make: *** [Release] Error 2
bernie@Bernie7:~/of_v0.9.0_linux64_release/examples/graphics/polygonExample$

--- a/scripts/linux/ubuntu/install_dependencies.sh
+++ b/scripts/linux/ubuntu/install_dependencies.sh
@@ -113,7 +113,7 @@ then
 	fi
 fi
 
-PACKAGES="freeglut3-dev libasound2-dev libxmu-dev libxxf86vm-dev g++${CXX_VER} libgl1-mesa-dev${XTAG} libglu1-mesa-dev libraw1394-dev libudev-dev libdrm-dev libglew-dev libopenal-dev libsndfile-dev libfreeimage-dev libcairo2-dev libfreetype6-dev libssl-dev libpulse-dev libusb-1.0-0-dev libgtk${GTK_VERSION}-dev  libopencv-dev libassimp-dev librtaudio-dev libboost-filesystem${BOOST_VER}-dev libgstreamer${GSTREAMER_VERSION}-dev libgstreamer-plugins-base${GSTREAMER_VERSION}-dev  ${GSTREAMER_FFMPEG} gstreamer${GSTREAMER_VERSION}-pulseaudio gstreamer${GSTREAMER_VERSION}-x gstreamer${GSTREAMER_VERSION}-plugins-bad gstreamer${GSTREAMER_VERSION}-alsa gstreamer${GSTREAMER_VERSION}-plugins-base gstreamer${GSTREAMER_VERSION}-plugins-good"
+PACKAGES="libjack-dev libjack0 libaudio4 freeglut3-dev libasound2-dev libxmu-dev libxxf86vm-dev g++${CXX_VER} libgl1-mesa-dev${XTAG} libglu1-mesa-dev libraw1394-dev libudev-dev libdrm-dev libglew-dev libopenal-dev libsndfile-dev libfreeimage-dev libcairo2-dev libfreetype6-dev libssl-dev libpulse-dev libusb-1.0-0-dev libgtk${GTK_VERSION}-dev  libopencv-dev libassimp-dev librtaudio-dev libboost-filesystem${BOOST_VER}-dev libgstreamer${GSTREAMER_VERSION}-dev libgstreamer-plugins-base${GSTREAMER_VERSION}-dev  ${GSTREAMER_FFMPEG} gstreamer${GSTREAMER_VERSION}-pulseaudio gstreamer${GSTREAMER_VERSION}-x gstreamer${GSTREAMER_VERSION}-plugins-bad gstreamer${GSTREAMER_VERSION}-alsa gstreamer${GSTREAMER_VERSION}-plugins-base gstreamer${GSTREAMER_VERSION}-plugins-good"
 
 echo "installing OF dependencies"
 echo "OF needs to install the following packages using apt-get:"

--- a/scripts/linux/ubuntu/install_dependencies.sh
+++ b/scripts/linux/ubuntu/install_dependencies.sh
@@ -11,7 +11,47 @@ fi
 
 if [ "$1" == "-y" ]; then
     FORCE_YES=-y
-fi 
+fi
+
+function installPackages {
+    for pkg in $@; do
+        echo "Installing ${pkg}"
+        dpkg-query -W -f=' ' ${pkg} 2> /dev/null
+        if [ $? -eq 0 ]; then 
+            echo "Already installed"
+        else
+            error="$(apt-get install --dry-run ${pkg})"
+            exit_code=$?
+            echo "$error" | grep Remv > /dev/null
+            if [ $? -eq 0 ]; then
+                apt-get install ${pkg}
+                exit_code=$?
+                if [ $exit_code != 0 ]; then
+                    echo "error installing ${pkg}, there could be an error with your internet connection"
+                    echo "if the error persists, please report an issue in github: http://github.com/openframeworks/openFrameworks/issues"
+                    exit $exit_code
+                fi
+            elif [ $exit_code -eq 0 ]; then
+                apt-get install ${pkg} > /dev/null
+                exit_code=$?
+                if [ $exit_code != 0 ]; then
+                    echo "error installing ${pkg}, there could be an error with your internet connection"
+                    echo "if the error persists, please report an issue in github: http://github.com/openframeworks/openFrameworks/issues"
+                    exit $exit_code
+                fi
+            else
+                echo "error installing ${pkg}"
+                echo $error
+                echo "this seems an error with your distribution repositories but you can also"
+                echo "report an issue in the openFrameworks github: http://github.com/openframeworks/openFrameworks/issues"
+                exit $exit_code
+            fi
+        fi
+    done
+    echo
+    echo "All packages were installed correctly"
+    echo
+}
 
 MAJOR_VERSION=$(lsb_release -r | cut -f2 -d: | cut -f1 -d. | sed "s/\t//g")
 MINOR_VERSION=$(lsb_release -r | cut -f2 -d: | cut -f2 -d.)
@@ -65,8 +105,7 @@ XTAG=$(dpkg -l |grep xserver-xorg-core|grep ii|awk '{print $2}'|sed "s/xserver-x
 if [ ! -z $XTAG ]
 then
 	read -p " installing OF dependencies with "${XTAG}" packages, confirm Y/N ? " -n 1 -r
-	if [[ $REPLY =~ ^[Yy]$ ]]
-	then
+	if [[ $REPLY =~ ^[Yy]$ ]]; then
 		echo
 		echo "installation of OF dependencies with "${XTAG}" packages confirmed"
 	else
@@ -74,23 +113,23 @@ then
 	fi
 fi
 
+PACKAGES="freeglut3-dev libasound2-dev libxmu-dev libxxf86vm-dev g++${CXX_VER} libgl1-mesa-dev${XTAG} libglu1-mesa-dev libraw1394-dev libudev-dev libdrm-dev libglew-dev libopenal-dev libsndfile-dev libfreeimage-dev libcairo2-dev libfreetype6-dev libssl-dev libpulse-dev libusb-1.0-0-dev libgtk${GTK_VERSION}-dev  libopencv-dev libassimp-dev librtaudio-dev libboost-filesystem${BOOST_VER}-dev libgstreamer${GSTREAMER_VERSION}-dev libgstreamer-plugins-base${GSTREAMER_VERSION}-dev  ${GSTREAMER_FFMPEG} gstreamer${GSTREAMER_VERSION}-pulseaudio gstreamer${GSTREAMER_VERSION}-x gstreamer${GSTREAMER_VERSION}-plugins-bad gstreamer${GSTREAMER_VERSION}-alsa gstreamer${GSTREAMER_VERSION}-plugins-base gstreamer${GSTREAMER_VERSION}-plugins-good"
+
 echo "installing OF dependencies"
-apt-get ${FORCE_YES} install freeglut3-dev libasound2-dev libxmu-dev libxxf86vm-dev g++${CXX_VER} libgl1-mesa-dev${XTAG} libglu1-mesa-dev libraw1394-dev libudev-dev libdrm-dev libglew-dev libopenal-dev libsndfile-dev libfreeimage-dev libcairo2-dev libfreetype6-dev libssl-dev libpulse-dev libusb-1.0-0-dev libgtk${GTK_VERSION}-dev  libopencv-dev libassimp-dev librtaudio-dev libboost-filesystem${BOOST_VER}-dev
-exit_code=$?
-if [ $exit_code != 0 ]; then
-    echo "error installing dependencies, there could be an error with your internet connection"
-    echo "if the error persists, please report an issue in github: http://github.com/openframeworks/openFrameworks/issues"
-	exit $exit_code
-fi
-
-
-echo "installing gstreamer"
-apt-get ${FORCE_YES} install libgstreamer${GSTREAMER_VERSION}-dev libgstreamer-plugins-base${GSTREAMER_VERSION}-dev  ${GSTREAMER_FFMPEG} gstreamer${GSTREAMER_VERSION}-pulseaudio gstreamer${GSTREAMER_VERSION}-x gstreamer${GSTREAMER_VERSION}-plugins-bad gstreamer${GSTREAMER_VERSION}-alsa gstreamer${GSTREAMER_VERSION}-plugins-base gstreamer${GSTREAMER_VERSION}-plugins-good
-exit_code=$?
-if [ $exit_code != 0 ]; then
-	echo "error installing gstreamer, there could be an error with your internet connection"
-    echo "if the error persists, please report an issue in github: http://github.com/openframeworks/openFrameworks/issues"
-	exit $exit_code
+echo "OF needs to install the following packages using apt-get:"
+echo ${PACKAGES}
+if [ "$1" != "-y" ]; then
+    read -p "Do you want to continue? [Y/n] "
+    if [[ $REPLY =~ ^[Nn]$ ]]; then
+        exit 0
+    fi
+    
+    echo
+    echo "Installing..."
+    echo
+    installPackages ${PACKAGES}
+else
+    installPackages ${PACKAGES}
 fi
 
 if [ $(expr $MAJOR_VERSION \< 13 ) -eq 1 ]; then

--- a/scripts/linux/ubuntu/install_dependencies.sh
+++ b/scripts/linux/ubuntu/install_dependencies.sh
@@ -113,7 +113,7 @@ then
 	fi
 fi
 
-PACKAGES="libjack-dev libjack0 libaudio-dev freeglut3-dev libasound2-dev libxmu-dev libxxf86vm-dev g++${CXX_VER} libgl1-mesa-dev${XTAG} libglu1-mesa-dev libraw1394-dev libudev-dev libdrm-dev libglew-dev libopenal-dev libsndfile-dev libfreeimage-dev libcairo2-dev libfreetype6-dev libssl-dev libpulse-dev libusb-1.0-0-dev libgtk${GTK_VERSION}-dev  libopencv-dev libassimp-dev librtaudio-dev libboost-filesystem${BOOST_VER}-dev libgstreamer${GSTREAMER_VERSION}-dev libgstreamer-plugins-base${GSTREAMER_VERSION}-dev  ${GSTREAMER_FFMPEG} gstreamer${GSTREAMER_VERSION}-pulseaudio gstreamer${GSTREAMER_VERSION}-x gstreamer${GSTREAMER_VERSION}-plugins-bad gstreamer${GSTREAMER_VERSION}-alsa gstreamer${GSTREAMER_VERSION}-plugins-base gstreamer${GSTREAMER_VERSION}-plugins-good"
+PACKAGES="curl libjack-dev libjack0 libaudio-dev freeglut3-dev libasound2-dev libxmu-dev libxxf86vm-dev g++${CXX_VER} libgl1-mesa-dev${XTAG} libglu1-mesa-dev libraw1394-dev libudev-dev libdrm-dev libglew-dev libopenal-dev libsndfile-dev libfreeimage-dev libcairo2-dev libfreetype6-dev libssl-dev libpulse-dev libusb-1.0-0-dev libgtk${GTK_VERSION}-dev  libopencv-dev libassimp-dev librtaudio-dev libboost-filesystem${BOOST_VER}-dev libgstreamer${GSTREAMER_VERSION}-dev libgstreamer-plugins-base${GSTREAMER_VERSION}-dev  ${GSTREAMER_FFMPEG} gstreamer${GSTREAMER_VERSION}-pulseaudio gstreamer${GSTREAMER_VERSION}-x gstreamer${GSTREAMER_VERSION}-plugins-bad gstreamer${GSTREAMER_VERSION}-alsa gstreamer${GSTREAMER_VERSION}-plugins-base gstreamer${GSTREAMER_VERSION}-plugins-good"
 
 echo "installing OF dependencies"
 echo "OF needs to install the following packages using apt-get:"

--- a/scripts/linux/ubuntu/install_dependencies.sh
+++ b/scripts/linux/ubuntu/install_dependencies.sh
@@ -32,7 +32,7 @@ function installPackages {
                     exit $exit_code
                 fi
             elif [ $exit_code -eq 0 ]; then
-                apt-get install ${pkg} > /dev/null
+                apt-get -y install ${pkg} > /dev/null
                 exit_code=$?
                 if [ $exit_code != 0 ]; then
                     echo "error installing ${pkg}, there could be an error with your internet connection"
@@ -113,7 +113,7 @@ then
 	fi
 fi
 
-PACKAGES="libjack-dev libjack0 libaudio4 freeglut3-dev libasound2-dev libxmu-dev libxxf86vm-dev g++${CXX_VER} libgl1-mesa-dev${XTAG} libglu1-mesa-dev libraw1394-dev libudev-dev libdrm-dev libglew-dev libopenal-dev libsndfile-dev libfreeimage-dev libcairo2-dev libfreetype6-dev libssl-dev libpulse-dev libusb-1.0-0-dev libgtk${GTK_VERSION}-dev  libopencv-dev libassimp-dev librtaudio-dev libboost-filesystem${BOOST_VER}-dev libgstreamer${GSTREAMER_VERSION}-dev libgstreamer-plugins-base${GSTREAMER_VERSION}-dev  ${GSTREAMER_FFMPEG} gstreamer${GSTREAMER_VERSION}-pulseaudio gstreamer${GSTREAMER_VERSION}-x gstreamer${GSTREAMER_VERSION}-plugins-bad gstreamer${GSTREAMER_VERSION}-alsa gstreamer${GSTREAMER_VERSION}-plugins-base gstreamer${GSTREAMER_VERSION}-plugins-good"
+PACKAGES="libjack-dev libjack0 libaudio-dev freeglut3-dev libasound2-dev libxmu-dev libxxf86vm-dev g++${CXX_VER} libgl1-mesa-dev${XTAG} libglu1-mesa-dev libraw1394-dev libudev-dev libdrm-dev libglew-dev libopenal-dev libsndfile-dev libfreeimage-dev libcairo2-dev libfreetype6-dev libssl-dev libpulse-dev libusb-1.0-0-dev libgtk${GTK_VERSION}-dev  libopencv-dev libassimp-dev librtaudio-dev libboost-filesystem${BOOST_VER}-dev libgstreamer${GSTREAMER_VERSION}-dev libgstreamer-plugins-base${GSTREAMER_VERSION}-dev  ${GSTREAMER_FFMPEG} gstreamer${GSTREAMER_VERSION}-pulseaudio gstreamer${GSTREAMER_VERSION}-x gstreamer${GSTREAMER_VERSION}-plugins-bad gstreamer${GSTREAMER_VERSION}-alsa gstreamer${GSTREAMER_VERSION}-plugins-base gstreamer${GSTREAMER_VERSION}-plugins-good"
 
 echo "installing OF dependencies"
 echo "OF needs to install the following packages using apt-get:"

